zoltar-core/src/main/java/com/spotify/zoltar/DefaultPredictorBuilder.java

```diff
@@ -128,32 +128,14 @@
 
   public abstract AsyncPredictFn<ModelT, InputT, VectorT, ValueT> predictFn();
 
-  public abstract Builder<ModelT, InputT, VectorT, ValueT> toBuilder();
+  public abstract Predictor<InputT, ValueT> predictor();
 
   @Override
-  public Predictor<InputT, ValueT> predictor(
+  public DefaultPredictorBuilder<ModelT, InputT, VectorT, ValueT> with(
```

Comment:
What's the point of this method now? Maybe I'm missing something but isn't this just a duplicate of `create()`?

Reply:
Like this is an instance method but just ends up returning a new one via `create`